transports/daily/src/transport.ts

```diff
@@ -475,7 +495,6 @@ export class DailyTransport extends Transport {
     this.stopRecording();
 
     await this._daily.leave();
-    await this._daily.destroy();
```

Comment:
I thought we had a release or destroy method where we could include this, but I was wrong. 🤕